FirebaseUI/Database/API/FirebaseArrayDelegate.h

```diff
@@ -36,7 +38,7 @@
  * @param object The object added to the FirebaseArray
  * @param index The index the child was added at
  */
-- (void)childAdded:(id)object atIndex:(NSUInteger)index;
+- (void)array:(FirebaseArray *)array didAddObject:(id)object atIndex:(NSUInteger)index;
```

Comment:
Renamed to follow Cocoa convention; now has more clear semantic of when the call happens (immediate after insertion, as opposed to a hypothetical `willAddObject:` method that could be added in the future).